### PR TITLE
fix(aapd-1524): use decimal for number component, fix task list

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/database/src/schema.prisma [hunk 1]
```diff
@@ -525,7 +525,7 @@ model AppellantSubmission {
   appellantPhoneNumber           String?
 
   // number entries
-  siteAreaSquareMetres Int?
+  siteAreaSquareMetres Float?
```
Comment (fabfab-kc) [inline packages/database/src/schema.prisma:528]: Hi @Si3rr4wow, I am putting together the JSON schema for the appeal data. In that format, `number` covers any numeric value (INT, FLOAT, DECIMAL, LONG), so currently I'm using `"multipleOf": 0.1` to indicate a single decimal point...

Do you know how the front office validates this value? I can align my side, or maybe a `Decimal?` type would work better on SQL?

Comment (Si3rr4wow) [reply packages/database/src/schema.prisma:528]: Hey Fabio, I was told we were making this change to align with you 😅 We still want this to be an integer on the front end

Comment (Si3rr4wow) [reply packages/database/src/schema.prisma:528]: Are you saying we can just leave this an an int and BO won't have a problem with that?

Comment (fabfab-kc) [reply packages/database/src/schema.prisma:528]: No I meant `Float -> Decimal` :-)
Let me check with @beejjacobs ...

Comment (Si3rr4wow) [reply packages/database/src/schema.prisma]: Oh sorry, done :)

File: packages/database/src/migrations/20240508131806_/migration.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[AppellantSubmission] ALTER COLUMN [siteAreaSquareMetres] FLOAT(53) NULL;
```
Comment (beejjacobs) [inline packages/database/src/migrations/20240508131806_/migration.sql:6]: Need to delete this migration and recreate with the decimal type; also be nice if the migration had a name :)

**Review by beejjacobs — CHANGES_REQUESTED** [review]
One lil comment

File: packages/forms-web-app/src/dynamic-forms/has-appeal-form/journey.js [hunk 3]
```diff
@@ -36,6 +36,8 @@ class HasAppealFormJourney extends Journey {
 			returnToListing: true
 		});
 
+		console.log('👷‍♀️', response.answers);
+
```
Comment (beejjacobs) [inline packages/forms-web-app/src/dynamic-forms/has-appeal-form/journey.js:40]: Left in from dev?

File: packages/database/src/migrations/20240508131806_/migration.sql (hunk 2 above)
Comment (Si3rr4wow) [reply packages/database/src/migrations/20240508131806_/migration.sql]: done and done :)

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**